WalletWasabi.Gui/Converters/IsNotNullOrEmptyBoolConverter.cs

```diff
@@ -0,0 +1,29 @@
+using Avalonia.Data.Converters;
+using System;
+using System.Globalization;
+
+namespace WalletWasabi.Gui.Converters
+{
+	internal class IsNotNullOrEmptyBoolConverter : IValueConverter
```

Comment:
This class could be unnecessary. Take a look at how we were doing it in other places with the `StringConverters.IsNotNullOrEmpty` converter.

Reply:
You are right! ACK